phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+    level: 5
```

Comment:
can we try 8?
all code, core and project should aim to that level

Reply:
Updated to level 8  
We have two types of PhpStan errors: 
 *  return type has no value type specified in iterable type Symfony\Component\Process\Process 
 * return type has no value type specified in iterable type array  

iterable type array - will fix in next story (when will implement valuable work with JSON files)